.github/workflows/docker_publish.yml

```diff
@@ -48,9 +48,26 @@ env:
   ECR_REPO_REGION: "us-east-1"
 
 jobs:
+  create-aarch64-runner:
```

Comment:
can we use a cpu runner instead?

Reply:
Sure. small question here: is there any specific reason we prefer the cpu one?

The graviton instance `c7g.4xlarge` is less than half of the cost of the CPU instance used here `c6i.8xlarge`.

Reply:
oh interesting. let's keep graviton then. graviton instance AMI doesn't come installed with docker and some other packages, so there's just some additional overhead with those.